File: backend/src/main/java/edu/cit/morre/campuscare/dto/ProfileResponse.java
package edu.cit.morre.campuscare.dto;

public class ProfileResponse {

    private String firstName;
    private String lastName;
    private String email;
    private String role;

    public ProfileResponse(String firstName, String lastName, String email, String role) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.email = email;
        this.role = role;
    }

    public String getFirstName() { return firstName; }
    public String getLastName() { return lastName; }
    public String getEmail() { return email; }
    public String getRole() { return role; }
}

File: mobile/app/src/main/java/edu/cit/morre/campuscare/model/ProfileResponse.kt
package edu.cit.morre.campuscare.model

data class ProfileResponse(
    val firstName: String,
    val lastName: String,
    val email: String,
    val role: String
)

data class UpdateProfileRequest(
    val firstName: String,
    val lastName: String
)

data class ChangePasswordRequest(
    val currentPassword: String,
    val newPassword: String,
    val confirmPassword: String
)